extensions/wikia/ImageReview/ImageCountGetter.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class ImageCountGetter extends WikiaModel {
```

Comment:
This class is responsible for getting the image counts (how many images are left in rejected/questionable/unreviewed). Unlike the previous implementation, it doesn't cache the results, it runs the query for each request which allows us to get rid of a bunch of tricky caching logic.

Based on [this dashboard](https://kibana.wikia-inc.com/index.html#dashboard/temp/AVi5YAviAinGJK4DXqYl) it doesn't look like the tool receives an unmanageable amount of traffic. In the last week it's gone up to 1188 requests in an hour, but usually is much lower than that.

Additionally, I tried running the 2 queries this uses on production and they seem (IMO) sufficiently performant for our use case here. About 0.03 and 0.01 respectively. Here are the explains:

```
mysql@slave.db-archive.service.consul[dataware]>explain SELECT COUNT( * ) AS total, state
    ->          FROM image_review
    ->          WHERE top_200 = '0'
    ->          AND state IN ( '5', '4', '0', '98' ) GROUP BY state;
+----+-------------+--------------+-------+------------------------+-------------+---------+------+--------+--------------------------+
| id | select_type | table        | type  | possible_keys          | key         | key_len | ref  | rows   | Extra                    |
+----+-------------+--------------+-------+------------------------+-------------+---------+------+--------+--------------------------+
|  1 | SIMPLE      | image_review | range | query_idx2,image_list2 | image_list2 | 6       | NULL | 100620 | Using where; Using index |
+----+-------------+--------------+-------+------------------------+-------------+---------+------+--------+--------------------------+
```
```
mysql@slave.db-archive.service.consul[dataware]>EXPLAIN SELECT COUNT( * ) AS total, state          FROM image_review          WHERE top_200 = '0'          AND state IN ( '6', '7' )          AND reviewer_id = '4929823' GROUP BY state;
+----+-------------+--------------+-------+--------------------------------------------+------------+---------+------+------+------------------------------------+
| id | select_type | table        | type  | possible_keys                              | key        | key_len | ref  | rows | Extra                              |
+----+-------------+--------------+-------+--------------------------------------------+------------+---------+------+------+------------------------------------+
|  1 | SIMPLE      | image_review | range | query_idx2,review_by_edit_prio,image_list2 | query_idx2 | 4       | NULL |    2 | Using index condition; Using where |
+----+-------------+--------------+-------+--------------------------------------------+------------+---------+------+------+------------------------------------+
1 row in set (0.01 sec)
```